.github/workflows/ci.yaml

```diff
@@ -107,21 +107,33 @@ jobs:
   operator-build:
     name: Check operator container image build
     runs-on: ubuntu-latest
+
+    env:
+      PUSH_IMAGE: false
```

Comment:
Can we separate out buildx image push from image build in the makefile targets? Then it would be consistent with the pre-existing `docker-build` and `docker-push` targets we have.

Reply:
Yeah I can separate it out if you think that'd be best, initially I did it this way with the env var because I thought it would save from repeating too much of the same code and made the cleanup easier. Also, since it is using a manifest to contain the tags for all the built images, it might need to work a bit differently than the typical single image build/push separation. 

I would probably need to do something like this to ensure there isn't any files or manifests left over:

```
docker-buildx-build: test ## Build and push docker image for the manager for cross-platform support
# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
	- docker buildx create --name registry-operator-builder
	docker buildx use registry-operator-builder
	- docker buildx build --platform=$(PLATFORMS) --tag ${IMG} --provenance=false -f Dockerfile.cross $(shell pwd)
	- docker buildx rm registry-operator-builder
	rm Dockerfile.cross

docker-buildx-push: test 
	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
	- docker buildx create --name registry-operator-builder
	docker buildx use registry-operator-builder
	- docker buildx build --push --platform=$(PLATFORMS) --tag ${IMG} --provenance=false -f Dockerfile.cross $(shell pwd)
	- docker buildx rm registry-operator-builder
	rm Dockerfile.cross
```


The `docker buildx rm registry-operator-builder` and `rm Dockerfile.cross` commands are important because this cleans up the added files and manifests used to create the multi-arch images, if I left those out of the `build` portion then if the `push` portion was never called (such as the case when we are only testing builds) they would be leftover, although i'm unsure what repercussions that would have on our workflows without investigating. On the flip side if I included them and then we had the `push` after, it wouldn't know what to push since the local manifest would've been deleted before `push` was called.

Do you think the above is too repetitive?

Reply:
I think that looks fine.